### PR TITLE
M rake closes #17 closes #3 closes #4 closes #12 closes #13

### LINKED ISSUE #3: Matt - Merchant Record Endpoint - Show

### LINKED ISSUE #4: Matt - Merchant Record Endpoint - Single find

### LINKED ISSUE #12: Matt - Customer Record Endpoint - index

### LINKED ISSUE #13: Matt - Customer Record Endpoint - Show

### LINKED ISSUE #17: Matt - Create Rake Task for seeding Customers, Invoice Items, Invoices, Items, Merchants, Transactions

### PR DESCRIPTION
**Pull request by mgmilton**
Description of Changes:
-
* Adds rake file for seeding the database
* Adds csv_import.md for documenting the usage of the rake file
* Adds customer, merchant, and transactions models 

### Tests Added:
* spec/requests/api/v1/customers_request_spec.rb
* spec/requests/api/v1/merchants_request_spec.rb

#### Reference Related Issue(s) or Spec Requirement:
- #17 
- #3 
- #4 
- #12
- #13 

#### Checklist:
- [x] Code follows code style of this project
- [X] Tests added to cover changes
- [X] All new and existing tests passing
#

#### Questions / Additional Notes:

@mgmilton @lnchambers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,6 @@
 /yarn-error.log
 
 .byebug_history
+
+/data
```
Comment (lnchambers) [inline .gitignore:25]: Let's revisit this later

File: app/models/customer.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Customer < ApplicationRecord
```
Comment (lnchambers) [inline app/models/customer.rb:1]: Can we add some validations here?

File: app/models/merchant.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class Merchant < ApplicationRecord
```
Comment (lnchambers) [inline app/models/merchant.rb:1]: And validations here

File: spec/models/merchant_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+# RSpec.describe Merchant, type: :model do
```
Comment (lnchambers) [inline spec/models/merchant_spec.rb:3]: Can we remove these comments and add shoulda matchers after doing the validations on the model?

File: spec/models/customer_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+# RSpec.describe Customer, type: :model do
+#   pending "add some examples to (or delete) #{__FILE__}"
```
Comment (lnchambers) [inline spec/models/customer_spec.rb:4]: See below

File: spec/requests/api/v1/transactions_request_spec.rb [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# require "rails_helper"
```
Comment (lnchambers) [inline spec/requests/api/v1/transactions_request_spec.rb:1]: I'm not a fan of comments in master. Let's try and keep it to a minimum

**Review by lnchambers — APPROVED** [review]
giphy 5

File: .gitignore (hunk 1 above)
Comment (mgmilton) [reply .gitignore:25]: Great catch with regard to cloning down the repo and using the data.

**Pull request merged**